.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -2,16 +2,32 @@
 name: Feature request
 about: Suggest an idea for this project
 title: ''
-labels: ''
+labels: 'feature'
 assignees: ''
 ---
 
-**Is your feature request related to a problem? Please describe.** A clear and concise description of what the problem
-is. Ex. I'm always frustrated when [...]
+## The feature
 
-**Describe the solution you'd like** A clear and concise description of what you want to happen.
+Briefly describe the feature you want to see added to Otomi, preferably in the form of a user-story (e.g. "As a [persona], 
+I [want to], [so that].") 
 
-**Describe alternatives you've considered** A clear and concise description of any alternative solutions or features
+## Related issues
+
+Please link all related issues in order to provide a good overview of the context.
+
+## Possible solution
+
+A clear and concise description of what you want to happen.
+
+## Alternative solutions
+
+A clear and concise description of any alternative solutions or features
 you've considered.
 
-**Additional context** Add any other context or screenshots about the feature request here.
+## Breaking stuff and backwards compatibility
+
+Will the new feature break current environments and support backwards compatibility? 
+
+## Additional context
+
+If applicable, add any other context about the problem here.
```

Comment:
Aren't first four items part of commit message?

feat: add new feature
BREAKING CHANGE: this is breaking change

OR

fix: change sth
BREAKING CHANGE: this is breaking change